### PR TITLE
Steffany Broken Rails Console :(

### PR DESCRIPTION
**Pull request by steffnay**
# API Muncher
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
:------------- | :-------------
How did you go about exploring the Edamam API, how did you try querying the API? | 
Describe your API Wrapper.  How did you decide on the methods you created? | 
Describe an edge case or failure case test you wrote for your API Wrapper. | 
Explain how VCR aids in testing an API. | 
What is the Heroku URL of your deployed application? | 
Provide a link to the Trello board you used |

### CODE REVIEW

File: app/controllers/recipe_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @recipes = results.paginate(:page => params[:page], :per_page => 10)
+
+    if @recipes.length > 0
+      flash[:success] = "#{@recipes.length} Recipes Found!"
+    else
+      flash[:alert] = "No Recipes Found :("
+    end
+  end
+
+  def create
```
Comment (CheezItMan) [inline app/controllers/recipe_controller.rb:15]: You have a lot of unused controller actions

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  root 'recipe#new'
+  get 'recipe/results', to: 'recipe#index', as: 'results'
```
Comment (CheezItMan) [inline config/routes.rb:3]: This should be more like an index path `/recipes`

**Comment by CheezItMan** [comment]
# API Muncher
## What We're Looking For

Feature | Feedback
:------------- | :-------------
**Core Requirements** | 
Git hygiene | You need to be making more commit, commit messages are good however
Comprehension questions | **MISING**
**General** | 
Rails fundamentals (RESTful routing, use of named paths) | Your search results path is **not** RESTful.  
Semantic HTML | It's odd the PR has fairly semantic code, but the one I found on your github account had a `nav` and `header` element outside the `body` tag.  
Errors are reported to the user | No errors reported if there are no search results and it crashes if the `uri` for the show action is wrong. 
API Wrapper to handle the API requests | Look good!
Controller testing | **MISSING**, as you noted
Lib testing | **MISSING**, as you noted
Search Functionality | Check
List Functionality | Check
Show individual item functionality (link to original recipe opens in new tab) | Does not open in a new tab, but that's a quibble. 
**Styling** | 
Responsive layout | Somewhat responsive.  The search results are responsive to a degree, but the show action's responsiveness needs work.
List View shows 10 items at a time/pagination | Nice work
The app is styled to create an attractive user interface | The app looks really good.  One note is that some of the text can be hard to read against the background image.   
**API Features** | 
The App attributes Edaman | Check
The VCR casettes do not contain the API key | Check
**External Resources** | 
Link to Trello Board | **MISSING**
Link to deployed app on Heroku | **MISSING**
**Overall** | I know you had trouble with Rails and given those issues this looks good.  As you noted testing is missing, and you need a bit more practice on responsive design.  You also need to make sure that errors are checked for in your controllers and you provide the user feedback.  Otherwise nice work.
